### PR TITLE
 payments transaction integration

### PR DESCRIPTION
**Pull request by seyiadel**
created a transaction base
retrieved data from webhook and add to database as transaction logs
implemented a route to get current user transactions

### CODE REVIEW

File: payment/models.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from config.database import Base
+from sqlalchemy import Column, Integer, String,Time,DateTime, ForeignKey
+from sqlalchemy.orm import relationship
+import uuid
+from sqlalchemy.dialects.postgresql import UUID
+from datetime import datetime
+
+class Transaction(Base):
+    __tablename__ = "transaction"
+    id = Column(UUID(as_uuid=True),primary_key=True, default=uuid.uuid4,unique=True)
```
Comment (madvirus-ops) [inline payment/models.py:10]: I don't think using uuid as pk will work, use normal id as pk, then uuid as the id

**Review by madvirus-ops — CHANGES_REQUESTED** [review]
Make changes to the pk

File: payment/models.py (hunk 1 above)
Comment (seyiadel) [reply payment/models.py:10]: It works. The downside is the memory and query performance that can be optimized

Comment (madvirus-ops) [reply payment/models.py:10]: Okay

**Review by madvirus-ops — APPROVED** [review]
Whatever you say 🤟

**Pull request merged**